prow/plugins/config.go

```diff
@@ -1804,3 +1805,20 @@ func (p *Plugins) mergeFrom(other *Plugins) error {
 
 	return utilerrors.NewAggregate(errs)
 }
+
+func (c *Configuration) HasConfigFor() (global bool, orgs sets.String, repos sets.String) {
+	if !reflect.DeepEqual(c, &Configuration{Plugins: c.Plugins}) {
+		global = true
```

Comment:
Not sure I follow - a config is global if it contains more than just plugins? Why? Since that's the only sub-section we've added sharding for thus far?

Reply:
Yeah, because the plugins are the only part we implemented merging for